src/components/ConsentPicker.vue

```diff
@@ -0,0 +1,486 @@
+<template>
+  <PvPanel class="m-0 p-0 w-full" header="Select params for the Consent and Assent forms">
+    <div class="card flex justify-content-center">
+      <div class="flex flex-wrap gap-3">
+        <div class="flex align-items-center">
+          <PvRadioButton v-model="decision" inputId="helpChoose" name="help" value="help" @change="whatDecision" />
+          <label for="helpChoose" class="ml-2">Help me choose</label>
+        </div>
+        <div class="flex align-items-center">
+          <PvRadioButton v-model="decision" inputId="iKnow" name="know" value="know" @change="whatDecision" />
+          <label for="iKnow" class="ml-2">I know what to select</label>
+        </div>
+      </div>
+    </div>
+    <div class="flex flex-row">
+      <div class="align-content-center" style="width: 50%" v-if="helpMeSelect">
+        <h3>Default Data Collection</h3>
+        <div class="border-solid border-round" style="width: 70%">
+          <div v-for="param in defaultParams" :key="param" class="mt-1 mb-1 ml-2 text-center flex">
+            <div class="ml-2 mr-3 flex" style="width: 80%">- {{ param.name }}</div>
+            <i :class="param.icon" style="font-size: 1rem; width: 20%"></i>
```

Comment:
Feel free to disregard this comment, but I feel that the icons might be better suited left aligned next to the header, as we do in the navbar dropdown menu. I feel that spacing the icon and header apart is incongruent with the function of the icon in this use case (providing context to the option). What do you think? 
<img width="204" alt="Screenshot 2024-06-04 at 2 44 07 PM" src="https://github.com/yeatmanlab/roar-dashboard/assets/20426569/2804d483-6e17-4656-9a86-4bf539281c89">

Reply:
Sure!